src/libraries/System.Reflection.Emit/tests/System.Reflection.Emit.Tests.csproj

```diff
@@ -115,4 +117,9 @@
     <Compile Include="TypeBuilder\TypeBuilderToString.cs" />
     <Compile Include="Utilities.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.Reflection.Emit.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.CodeDom\src\System.CodeDom.csproj" />
```

Comment:
Do we really need `System.CodeDom`?

Reply:
Used for creating `TempFileCollection` in the constructor of test type `AssemblySaveTestsWithVariousTypes`, not sure if it is the preferred/common way for creating temp files in runtime, will check.
```cs
    public AssemblySaveTestsWithVariousTypes()
    {
        const bool keepFiles = true;
        Directory.CreateDirectory("testDir");
        TempFileCollection  tfc = new TempFileCollection("testDir", false);
        _fileLocation = tfc.AddExtension("dll", keepFiles);
    }
```

Reply:
We have src\libraries\common\src\System\IO\TempFileCollection.cs to that.

Reply:
Can this use `src\libraries\Common\tests\System\IO\TempFile.cs` test helper that we use everywhere else?